### DISCUSSION TITLE
Proposal: Add info about PHP process to the README

### DISCUSSION THREAD

**Discussion by sebastiandeppisch** [qruto/laravel-wave#60] [Ideas]
I tried the package with the Laravel dev server `php artisan serve` and was confused by the fact that my app no longer responded in the browser. But the dev server has only 1 HTTP connection by default. 

This can be changed via ENV:
Just add e.g. 
PHP_CLI_SERVER_WORKERS=10
to .env
https://www.php.net/manual/en/features.commandline.webserver.php

It probably makes general sense to mention in the README that you have to change this with artisan serve or that you have to consider the number of workers independently of the delployments.

**Comment by slavarazum** [comment]
Hey @sebastiandeppisch 👋 
Nice addition! I'll take a deeper look at this, maybe it will be possible to implement it in a more automatic way.

**Comment by slavarazum** [comment]
Starting from version 0.9.0, when you run `php artisan serve`, the command suggests setting `PHP_CLI_SERVER_WORKERS` for the current session and asks you to save this count for future use.
